goplaces/src/main/java/goplaces/apis/WaypointsReviewAPI.java

```diff
@@ -16,6 +16,8 @@
  * @author Aviral Takkar
  *
  * TODO Document the curl command to access API using POST request.
+ *
+ *
  */
```

Comment:
Yep try this:
"http://go-places-ucsb.appspot.com/enqueue_boxrouteworker_task?route_id=6473924464345088&radius=5000&keywords=pet%20park,museum"
The space should be encoded as %20 (I think) and I saw a double slash after appspot.com, shouldn't it be just a single slash?